perf/lua/CMakeLists.txt

```diff
@@ -61,7 +61,7 @@ function(create_perf_lua_test)
   )
   set_tests_properties(${test_title} PROPERTIES
     ENVIRONMENT "LUA_PATH=${LUA_PATH}"
-    LABELS "${TEST_SUITE_NAME};performance"
+    LABELS "${TEST_SUITE_NAME};performance;long"
```

Comment:
Nit: It is in a commit that relates to fuzzing tests. It seems, it should be in the commit that is about the performance tests.

Reply:
Moved hunk to an appropriate commit.